app/routes.py

```diff
@@ -178,6 +181,24 @@ def remove_user(email):
         print("Failed to remove user or user not found.")
 
 
+## Org management
+@org.cli.command("add")
+@click.argument("name")
+@click.option("--logo", default="", help="Org Logo")
+@click.option("--id", default="", help="Org ID: should correspond to CKAN ORG ID")
+def cli_add_org(name, logo, id):
```

Comment:
This looks good to me, but to be clear since these parameters are positional if any parameters are not included a `missing required positional arguments` error will be thrown. The `if <arg>` checks below will only check that they have a truthy value not that they exist in scope. Just want to make sure that that is the intention here.

Reply:
`name` is the only required field

`.argument` is positional
`.option` is optional / kwarg

https://click.palletsprojects.com/en/8.1.x/options/#basic-value-options